lib/plugins/tag/code.js

```diff
@@ -24,7 +24,7 @@ const rCaption = /\S[\S\s]*/;
  * @param {Object} mark Line highlight specific line(s), each value separated by a comma. Specify number range using a dash
  * Example: `mark:1,4-7,10` will mark line 1, 4 to 7 and 10.
  * @param {Object} wrap Wrap the code block in <table>, value must be a boolean
- * @returns {String} Code snippet with code highlighting
+ * @returns {Object} Code snippet with code highlighting
```

Comment:
The `codeTag` function does return `String`.

Reply:
thanks, I undo it ;)